examples/plugins/CMakeLists.txt

```diff
@@ -4,4 +4,6 @@
 #------------------------------------------------------------------------------#
 
 add_subdirectory(engine)
-add_subdirectory(operator)
+if(ADIOS2_HAVE_Sodium)
+  add_subdirectory(operator)
+endif()
```

Comment:
We already test for this in the operator cmakelists

Reply:
We do, but if we don't build the operator the example will be built, we can not use the example.